central/networkgraph/service/service.go

```diff
@@ -47,7 +47,7 @@ func newService(store nfDS.ClusterDataStore,
 	graphConfigDS datastore.DataStore) *serviceImpl {
 	return &serviceImpl{
 		clusterFlows:     store,
-		entities:         entities,
+		entityDS:         entities,
```

Comment:
nit:  While making this change, it might be cleaner to update the parameter name too.

Reply:
Done